etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messages/SendMessageHelper.java

```diff
@@ -21,67 +21,66 @@ public static SendMessageHelper getInstance() {
 
     private SendMessageHelper() {}
 
-    public void savePartnerMetadataForReceivedMessage(PartnerMetadata partnerMetadata) {
-        if (partnerMetadata.receivedSubmissionId() == null) {
+    public void savePartnerMetadataForInboundMessage(PartnerMetadata partnerMetadata) {
+        if (partnerMetadata.inboundReportId() == null) {
             logger.logWarning(
-                    "Received submissionId is null so not saving metadata for received message");
+                    "inboundReportId is null so not saving metadata for received message");
             return;
         }
         try {
-            partnerMetadataOrchestrator.updateMetadataForReceivedMessage(partnerMetadata);
+            partnerMetadataOrchestrator.updateMetadataForInboundMessage(partnerMetadata);
         } catch (PartnerMetadataException e) {
             logger.logError(
-                    "Unable to save metadata for receivedSubmissionId "
-                            + partnerMetadata.receivedSubmissionId(),
+                    "Unable to save metadata for inboundReportId "
+                            + partnerMetadata.inboundReportId(),
                     e);
         }
     }
 
-    public void saveSentMessageSubmissionId(String receivedSubmissionId, String sentSubmissionId) {
-        if (sentSubmissionId == null || receivedSubmissionId == null) {
+    public void saveReportIds(String inboundReportId, String outboundReportId) {
```

Comment:
I would rename this method as `saveOutboundReportId` because we are not saving the inbound report ID here.  Yes, we're passing in the inbound report ID, but that's only to get the correct metadata so we can save the sent report ID.

Reply:
In the function above this one, should the naming match this one `saveInboundReportId` or should the method you called out match `savePartnerMetadataForInboundMessage`

Reply:
I'd keep it as is, `savePartnerMetadataForInboundMessage`, because it saves the entire partner metadata object.  This method that this thread of comments is on only updates the outbound report ID on an existing.